### PR TITLE
Use the service account provided by the user when fetching data

### PR DESCRIPTION
**Pull request by andreeanica16**

### CODE REVIEW

File: src/main/java/com/google/sps/ProjectCenter.java [hunk 1]
```diff
@@ -34,16 +38,9 @@
...
-      // and by built-in service accounts when running on GAE, GCE or GKE.
-      credential = GoogleCredential.getApplicationDefault();
-      
-      // The createScopedRequired method returns true when running on GAE or a local developer
-      // machine. In that case, the desired scopes must be passed in manually. When the code is
-      // running in GCE, GKE or a Managed VM, the scopes are pulled from the GCE metadata server.
-      // See https://developers.google.com/identity/protocols/application-default-credentials for more information.
+      this.projectId = projectId;
+        
+      credential = GoogleCredential.fromStream(new FileInputStream("bt-dataflow-sql-demo.json"));
```
Comment (tblanshard) [inline src/main/java/com/google/sps/ProjectCenter.java:43]: Could "bt-dataflow-sql-demo.json" perhaps be passed in as a variable similar to projectId?

Comment (andreeanica16) [reply src/main/java/com/google/sps/ProjectCenter.java]: # #https://github.com/googleinterns/step229-2020/pull/11/commits/74e7cee4f7154b26907db90b39558b7a9b67c0a6

Comment (andreeanica16) [reply src/main/java/com/google/sps/ProjectCenter.java]: https://github.com/googleinterns/step229-2020/pull/11/commits/74e7cee4f7154b26907db90b39558b7a9b67c0a6

Comment (andreeanica16) [reply src/main/java/com/google/sps/ProjectCenter.java]: Thank you for your suggestion!

**Pull request merged**